storage.go

```diff
@@ -366,49 +371,105 @@ func (f *file) Move(fromPath string, toPath string) FileResponse {
 	return response
 }
 
-// CreatSignedUrl create a signed url for a file object
-func (f *file) CreatSignedUrl(filePath string, expiresIn int) SignedUrlResponse {
-	_json, _ := json.Marshal(map[string]interface{}{
+func (f *file) CreateSignedURLForUpload(filePath string, expiresIn int) (*SignedURLForUploadResponse, error) {
+	reqBody, err := json.Marshal(map[string]interface{}{
```

Comment:
```suggestion
	reqBody, err := json.Marshal(map[string]any{
```

Reply:
If it is already everywhere in the code base, leave it as it is

Reply:
Sadly everywhere :/